index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- CSS only-->
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+      crossorigin="anonymous"
+    />
+    <title>microverse survey</title>
+  </head>
+  <body>
+    <div class="container-fluid">
+      <h1 id="title">Microverse collaboration survey form</h1>
+      <p id="description">
+        This is a survey for the Microverse international applicants who are
+        working on various projects in our Microverse trials
+      </p>
+
+      <div class="container">
```

Comment:
According to best practices, it would be better to use a section or main tag with class container here.